frontend/device/pocketbook/device.lua

```diff
@@ -57,24 +57,54 @@ local PocketBook = Generic:new{
 }
 
 function PocketBook:init()
+
+    self.screen = require("ffi/framebuffer_mxcfb"):new{device = self, debug = logger.dbg}
+    self.powerd = require("device/pocketbook/powerd"):new{device = self}
+    self.input = require("device/input"):new{
+        device = self,
+        event_map = {
+            [KEY_MENU] = "Menu",
```

Comment:
I also see a power button in some of them?

Reply:
Yes. I removed it because judging from ``libinkview.h`` there is no Key-Event with this number (1002) so it should never be returned by the ``device/input-handlers``. I also found no reference in koreader anywhere else. 

Pressing the power button will not emit a Key-Event (in inkview-terms speaking) but EVT_BACKGROUND and therefore (also with the old code) there is no use for this key-definition.  Maybe @chrox can clear this up.